WcaOnRails/app/models/competition.rb

```diff
@@ -567,7 +568,15 @@ def has_base_entry_fee?
   end
 
   def has_fees?
-    base_entry_fee_lowest_denomination + competition_events.sum(:fee_lowest_denomination) > 0
+    if base_entry_fee_lowest_denomination.nil?
```

Comment:
I think we can re-implement this method as simply:

```
def has_fees?
  !base_entry_fee_lowest_denomination.nil?
```

I checked, and there are no competitions in our database that have a base_entry_fee equal to $0 *and* have fees set for competition events:

```
select Competitions.id, Competitions.base_entry_fee_lowest_denomination, competition_events.fee_lowest_denomination from Competitions
LEFT JOIN competition_events ON Competitions.id=competition_events.competition_id
where Competitions.base_entry_fee_lowest_denomination=0 AND competition_events.fee_lowest_denomination>0
```

This is because we don't actually have a UI for setting competition event fees yet :p

Reply:
Ok, I can change it, and once fees per event are implemented someone will take care of this I guess :P

Reply:
OH NEVERMIND SORRY I GOT CONFUSED.

Reply:
I think what you did here already is great.

Reply:
Not changing anything then :)